packages/snaps-controllers/src/test-utils/registry.ts

```diff
@@ -0,0 +1,15 @@
+import { SnapRegistry, SnapRegistryStatus } from '../snaps';
+
+export class MockSnapRegistry implements SnapRegistry {
+  get = jest.fn().mockImplementation(async (snaps) => {
+    return Promise.resolve(
```

Comment:
This could be `.mockResolvedValue`.

Reply:
I don't think it can since it needs the input?

Reply:
Ah sorry, you're right.